common/include/assign_port.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include "verilated.h"
+#include "verilog_int.h"
+
+// assign port with privimite
+void assign_port(CData &dest, const uint64_t src) { dest = src; }
```

Comment:
How about make it more strict?
void assign_port(CData &dest, const CData src) { dest = src; }

Reply:
Maybe accept the u64 first, and see if we meet any problem in the future? I think it is more convenient that user can assign any primitive type to the port.
I add `static_cast` to truncate the extra bits.